mmcv/runner/hooks/lr_updater.py

```diff
@@ -614,3 +615,223 @@ def format_param(name, optim, param):
         if name not in param:
             raise KeyError(f'{name} is not found in {param.keys()}')
         return param[name]
+
+
+@HOOKS.register_module()
+class ReduceLrUpdateHook(LrUpdaterHook):
+    """ReduceLROnPlateau Scheduler.
+
+    Reduce learning rate when a metric has stopped improving. This scheduler
+    reads a metrics quantity and if no improvement is seen for a 'patience'
```

Comment:
and the learning rate will be reduced if no improvement is observed for a ``patience`` number of epochs.